events/message.js

```diff
@@ -40,6 +40,12 @@ module.exports = async (client, message) => {
   // using this const varName = thing OR otherthign; is a pretty efficient
   // and clean way to grab one of 2 values!
   if (!cmd) return;
+  
+  //A command might be not enabled. This message informs the command invoker that the command is disabled.
+
+  if (cmd && !cmd.conf.enabled) {
```

Comment:
`cmd` cannot be falsey here since it was checked on line 42. It's redundant to check again.